Neuron/Sections/Dapp/WebView/AdvancedViewController.swift

```diff
@@ -98,13 +98,13 @@ class AdvancedViewController: UIViewController {
 
     @IBAction func sureAdvanceButton(_ sender: UIButton) {
         guard let inputGasPrice = inputGasPrice else {
-            Toast.showToast(text: "请输入GasPrice")
+            Toast.showToast(text: "DApp.Advanced.EmptyGasPrice".localized())
             return
         }
         let finalGasPrice = Web3Utils.parseToBigUInt(inputGasPrice, units: .Gwei)!
         if finalGasPrice < gasPrice {
             let gasPriceString = Web3Utils.formatToEthereumUnits(gasPrice, toUnits: .Gwei, decimals: 4, fallbackToScientific: true) ?? "0"
-            Toast.showToast(text: "请输入的GasPrice不小于\(gasPriceString) Gwei")
+            Toast.showToast(text: "DApp.Advanced.MinGasPrice".localized() + "\(gasPriceString) Gwei")
```

Comment:
Use a while sentence here so we can use `%@` or other like:

`String(format: "Gas price should not be less than %@ Gwei", "20") `

The reason is different languages will have different word order/format when handling numbers.